ios-base/Makefile

```diff
@@ -11,6 +11,10 @@ xcode:
 	mint run xcodegen
 	bundle exec pod install
 
+swiftgen:
+	mkdir -p $(GIT_REPO)/ios-base/DroidKaigi\ 2020/Generated
+	mint run swiftgen swiftgen
```

Comment:
Could you please `SwiftGen/SwiftGen@6.1.0` add to `Mintfile` ?

Reply:
Oh, sorry. I missed..
I will add it 👍